prow/config/config.yaml

```diff
@@ -159,6 +159,140 @@ branch-protection:
                   - "idc-jenkins-ci-tidb/check_release_note"
                   - "license/cla"
                 strict: true
+        docs:
+          branches:
+            master:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "ci/circleci: lint"
+                  - "license/cla"
+                strict: true
```

Comment:
It's should be fasle. Because I don't think there is much code in the documentation repository, this doesn't pay off and only blocked the pr merge.

Reply:
We do require base to be up to date though...
You can only imagine how many conflicts we resolve every day.😢

Reply:
I'm not sure what `strict: true/false` means. Can you explain it?

Reply:
If set to true, it will ask PR to update the latest base, if not false, there is no such request.

Reply:
OK, thanks. I'm fine with `false`.

Reply:
Updated.